app/services/analytics_events.rb

```diff
@@ -4035,14 +4047,16 @@ def user_registration_phone_setup_visit(enabled_mfa_methods_count:, **extra)
   end
 
   # Tracks when user's piv cac is disabled
-  def user_registration_piv_cac_disabled
-    track_event('User Registration: piv cac disabled')
+  def piv_cac_disabled
+    track_event('Piv Cac disabled')
   end
 
   # Tracks when user's piv cac setup
```

Comment:
For what it's worth, I did a bit of light poking around `identity-devops` and the ad hoc dashboards to get a sense if this would have any impact, and my impression is that it's probably reasonably-safe. I also like avoiding the 'User registration' bit if it's something we log outside the registration flow.